website/source/docs/secrets/aws/index.html.md

```diff
@@ -37,6 +37,10 @@ $ vault write aws/config/root \
     region=us-east-1
 ```
 
+*Note that `root` does not mean it needs to be your AWS account's root credentials, and it 
+probably should not be.  Create an IAM user with permissions to manage IAM and STS.  
```

Comment:
Maybe add something like: "You can also just launch Vault in an IAM instance profile with these permissions and you won't need to configure these AWS credentials.